samples/lib/Common.c

```diff
@@ -0,0 +1,980 @@
+#define LOG_CLASS "WebRtcSamples"
+#include "../Samples.h"
+
+PSampleConfiguration gSampleConfiguration = NULL;
+
+STATUS terminate(UINT32 timerId, UINT64 currentTime, UINT64 customData)
+{
+    UNUSED_PARAM(timerId);
+    UNUSED_PARAM(currentTime);
+    UNUSED_PARAM(customData);
+    DLOGI("Terminating the app");
+    if (gSampleConfiguration != NULL) {
+        ATOMIC_STORE_BOOL(&gSampleConfiguration->interrupted, TRUE);
+        ATOMIC_STORE_BOOL(&gSampleConfiguration->appTerminateFlag, TRUE);
+        CVAR_BROADCAST(gSampleConfiguration->cvar);
+    }
+    return STATUS_SUCCESS;
+}
+
+VOID sigintHandler(INT32 sigNum)
+{
+    UNUSED_PARAM(sigNum);
+    if (gSampleConfiguration != NULL) {
+        ATOMIC_STORE_BOOL(&gSampleConfiguration->interrupted, TRUE);
+        CVAR_BROADCAST(gSampleConfiguration->cvar);
+    }
+}
+
+STATUS signalingCallFailed(STATUS status)
+{
```

Comment:
If this isn't expected to be modified by customers later, would it be more appropriate to move it to `SignalingMsgHandler` and rename the file to something else?

Reply:
Retained here since it isnt a callback. SignalingMsgHandler is specific to SDP message handling.